### ISSUE TITLE
Ability to copy selected text

### ISSUE THREAD

**Issue by invakid404** [react-syntax-highlighter/react-syntax-highlighter#416]
**Is your feature request related to a problem? Please describe.**
I'm actually very surprised that this is not a thing, as it's something that sounds super essential to me: actually being able to copy the text you've **selected**.

**Describe the solution you'd like**
I would like to be able to actually properly copy the text I've selected, without it somehow including line numbers and random spaces and line breaks.

**Describe alternatives you've considered**
As mentioned in various other issues, related to copying text, a potential solution is to provide a separate way to copy the content, i.e. via a button or by overriding onCopy, but that doesn't really solve my problem. As an end user, I'd expect that I have the ability to selectively copy text, copying the full content can be extra annoying, especially if you're trying to copy a single file out of a massive file...

**Comment by nihirv** [comment]
Are you using Safari?

**Comment by invakid404** [comment]
No, I use Vivaldi, which is a Chromium-based browser. Your question actually inspired me to try in Firefox, and to my surprise, it works there. Not quite sure what's going on...

**Comment by invakid404** [comment]
Update: tried in Google Chrome as well, same issue, so this appears to be a thing in (most) Chromium-based browsers.

**Comment by nihirv** [comment]
On Edge and Chrome (both Chromium based) the copying seems to work fine for me. Sounds like you're on Linux?

**Comment by invakid404** [comment]
> On Edge and Chrome (both Chromium based) the copying seems to work fine for me. Sounds like you're on Linux?

Yeah, but a friend of mine who uses Chrome on Windows told me he has the same issue. For the record, I have wrap long lines and line numbers enabled.
